docs/engage/front-end-ui/buyback.md

```diff
@@ -0,0 +1,33 @@
+---
+title: BuyBack Module
+sidebar_position: 4
+---
+
+The BuyBack module provides full transparency into Defactor’s deflationary token strategy. It allows users to track buyback activity where ecosystem revenue is used to repurchase $FACTR tokens from the open market.
```

Comment:
Buyback is used by defactor but it can also be used by any client who want to , its not a defactor specific feature.